code/game/objects/items/weapons/material/shards.dm

```diff
@@ -18,6 +18,7 @@
 	applies_material_name = TRUE
 	unbreakable = 1 //It's already broken.
 	item_flags = ITEM_FLAG_CAN_HIDE_IN_SHOES
+	atom_flags = 0
```

Comment:
why dis

Reply:
ATOM_FLAG_NO_TEMPERATURE_CHANGE or whatever the macro is prevents temperature stuff from being called and is set by default.